service/telemetry.go

```diff
@@ -57,7 +57,7 @@ const (
 )
 
 type collectorTelemetryExporter interface {
-	init(col *Collector) error
+	init(srv *service) error
```

Comment:
nit: `svc` makes more sense to me here, as `srv` is associated with server, and not service. not a huge deal though.